### ISSUE TITLE
Exception 

### ISSUE THREAD

**Issue by ypiguet-epfl** [aseba-community/dashel#25]

**Comment by stephanemagnenat** [comment]
If critical system resources necessary for the functioning of Dashel, such as the termination pipes, cannot be acquired, the program should quit. These will happen only in case of system severely over-loaded to the point of being totally dysfunctional.

**Comment by ypiguet-epfl** [comment]
Then it should be clearly documented, so that potential users of the library can make an informed decision. There are applications where a communication error shouldn't be fatal.

**Comment by stephanemagnenat** [comment]
Yes I agree that this design choice should be documented. I thus relabel the issue as "enhancement" and change the title.
